package.json

```diff
@@ -23,10 +23,9 @@
     "test:size": "bundlesize"
   },
   "dependencies": {
-    "-ruslan-pdfkit": "git://github.com/RuslanNikolov/-ruslan-pdfkit",
+    "pdfkit": "git://github.com/RuslanNikolov/pdfkit.git",
```

Comment:
This needs to be changed back to the original @react-pdf/pdfkit once the changes are accepted.